content/sensu-go/6.0/operations/maintain-sensu/upgrade.md

```diff
@@ -11,7 +11,48 @@ menu:
     parent: maintain-sensu
 ---
 
-## Upgrade to the latest version of Sensu Go from 5.0.0 or later
+## Upgrade to Sensu Go 6.0 from a 5.x deployment
+
+{{% notice warning %}}
+**IMPORTANT**: Before you upgrade to Sensu 6.0, use [`sensuctl dump`](../../../sensuctl/create-manage-resources/#export-resources) to create a backup of your existing installation.
+You will not be able to downgrade to a Sensu 5.x version after you upgrade your database to Sensu 6.0 in step 3 of this process.
+{{% /notice %}}
+
+To upgrade your Sensu Go 5.x deployment to 6.0:
+
+1. [Install][1] the 6.0 packages or Docker image.
+2. Restart the services.
+   
+   {{% notice note %}}
+   **NOTE**: For systems that use `systemd`, run `sudo systemctl daemon-reload` before restarting the services.
+   {{% /notice %}}
+   
+   {{< code shell >}}
+# Restart the Sensu agent
+sudo service sensu-agent restart
+
+# Restart the Sensu backend
+sudo service sensu-backend restart
+{{< /code >}}
+
+3. Run a single upgrade command on one your Sensu backends to migrate the cluster:
+
+   {{< code shell >}}
+sensu-backend upgrade
+{{< /code >}}
+
+   {{% notice note %}}
+   **NOTE**: If you are deploying a new Sensu 6.0 cluster rather than upgrading from 5.x, you do not need to run the `sensu-backend upgrade` command.
+   {{% /notice %}}
+
+4. Enter `y` or `n` to confirm and wait a few seconds for the upgrade command to run.
+
+Until your cluster finishes upgrading, your entity list may return empty and proxy check requests may not work.
```

Comment:
A generic statement about the entity store would be "you may notice some inconsistencies with your entity list until the cluster is upgraded." I'm not sure if I've witnessed migration issues other than proxy entities but I could be wrong. Wouldn't want to leave out use cases either!